cwltool/singularity.py

```diff
@@ -89,6 +89,8 @@ def _normalize_sif_id(string: str) -> str:
 
 
 class SingularityCommandLineJob(ContainerCommandLineJob):
+    CONTAINER_TMPDIR: str = "/tmp"  # nosec
```

Comment:
Should `CONTAINER_TMPDIR` move into `ContainerCommandLineJob`?

Reply:
maybe?

Reply:
Yes it can; I will update